docs/output-elastic_workplace_search.asciidoc

```diff
@@ -140,7 +142,8 @@ To keep the timestamp field, set this value to the name of the field where you w
 * Value type is <<string,string>>
 * There is no default value
 
-The value of the API endpoint in the form of a URL.
+The value of the API endpoint in the form of a URL. It could be an on-premise instance like "http://workplace.company.com:3002",
+or an Elastic cloud instance, for example "https://7c455f508468426cb53912be65548117.ent-search.eu-west-1.aws.cloud.es.io"
```

Comment:
Good examples are always a nice doc improvement. Thank you for adding these.

What you have here builds and renders cleanly.  I would like to offer a suggestion to formatting for your consideration. 

The current formatting renders like this:
<img width="811" alt="Screen Shot 2022-01-05 at 8 59 22 AM" src="https://user-images.githubusercontent.com/35154725/148229683-f3a3ef28-b7a1-48d7-9e03-ce6372897b56.png">

The suggested formatting renders like this. (It looks better on the page than it does in this snippet.)  
<img width="836" alt="Screen Shot 2022-01-05 at 8 56 35 AM" src="https://user-images.githubusercontent.com/35154725/148229545-0fa371a6-a2d5-4ef7-8992-b261e117fe39.png">

I think that the expanded example makes it easier to quickly scan and extract the information. Both are correct. Since you are the author, please pick the format you like best. :-)

Reply:
Thanks a lot @karenzone for your always precious suggestions